src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -88,6 +92,9 @@ public class DrivetrainSubsystem extends SubsystemBase {
         private BooleanTopic rotationTargetTopic = table.getBooleanTopic("Has Rotation Target");
         private BooleanPublisher hasRotationTarget = rotationTargetTopic.publish(); 
 
+        private Field2d field = new Field2d();
+        private double simRotation = 0;
```

Comment:
Lets try to organize all of the odometry related variables into a common place in our declaration order